packages/react-dom-bindings/src/client/ReactDOMComponent.js

```diff
@@ -1809,6 +1814,7 @@ export function updatePropertiesWithDiff(
       const type = nextProps.type;
       const value = nextProps.value;
       const defaultValue = nextProps.defaultValue;
+      const oldDefaultValue = lastProps.defaultValue;
```

Comment:
Nit: `lastDefaultValue`. We use the term `last` instead of `old` everywhere else here.